<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="This is meta description for portfolio page">
<meta name="author" content="Somrat">










<title>Microarray</title>

<meta name="generator" content="Hugo 0.59.0" />


    <link rel="stylesheet" href="https://techne316.github.io/css/owl.carousel.css" />
    <link rel="stylesheet" href="https://techne316.github.io/css/bootstrap.min.css" />
    <link rel="stylesheet" href="https://techne316.github.io/css/font-awesome.min.css" />
    <link rel="stylesheet" href="https://techne316.github.io/css/ionicons.min.css" />
    <link rel="stylesheet" href="https://techne316.github.io/css/animate.css" />
    <link rel="stylesheet" href="https://techne316.github.io/css/syntax.css" />
    <link rel="stylesheet" href="https://techne316.github.io/css/venobox.css" />
<!-- Main Stylesheet -->

<link rel="stylesheet" href="https://techne316.github.io/css/style.min.css" integrity="" media="screen">



<script src="https://techne316.github.io/js/jquery-1.10.2.min.js"></script>
<script src="https://techne316.github.io/js/bootstrap.min.js"></script>
<script src="https://techne316.github.io/js/owl.carousel.min.js"></script>
<script src="https://techne316.github.io/js/plugins.js"></script>
<script src="https://techne316.github.io/js/venobox.min.js"></script>


<!-- Main Script -->

<script src="https://techne316.github.io/js/main.min.js"></script>

<!-- google analitycs -->
<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    ga('create', 'Your ID', 'auto');
    ga('send', 'pageview');
</script>



</head>
<body>

	
<header>
  <div class="container">
    <div class="row">
      <div class="col-md-12">

        
        <nav class="navbar navbar-default">
          <div class="container-fluid">
            
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
              </button>
              <a class="navbar-brand" href="https://techne316.github.io/">
                <img src="https://techne316.github.io/img/Techne.png" alt="Techne homepage Logo" style="width:210px;height:55px;">
              </a>
            </div>

            
              <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav navbar-right">
                  <li><a href="https://techne316.github.io/"> Home </a>
                  </li>
                  
                  
                  <li class="nav-item dropdown">
                    <a class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                      Project
                    </a>
                    <div class="dropdown-menu" >
                      
                      <a class="dropdown-item" href="/microarray">Microarray</a>
                      
                      <a class="dropdown-item" href="/rnaseq">RNAseq</a>
                      
                    </div>
                  </li>
                  
                  
                  
                  <li class="nav-item">
                    <a class="nav-link" href="/blog">Blog</a>
                  </li>
                  
                  
                  
                  <li class="nav-item">
                    <a class="nav-link" href="/service">Service</a>
                  </li>
                  
                  
                  
                  <li class="nav-item">
                    <a class="nav-link" href="/contact">Contact</a>
                  </li>
                  
                  
                </ul>
               </div> 
          </div>
        </nav> 
               <div class ="row">
                 <div class="col-md-10"></div>
                 <div class="col-md-2">
                  <ul class="list-inline lang-list">
                  
                  <li class="list-inline-item"><a href="https://techne316.github.io/kor/" style="text-align:right;font-size:60%;">KOR</a></li>
                  
                  <li class="list-inline-item"><a href="https://techne316.github.io/" style="text-align:right;font-size:60%;">ENG</a></li>
                  
                  </ul>
                 </div> 

      </div>
    </div>
  </div>

</header>
	
	
		
<!DOCTYPE html>
<html>
  <head>
    
<section id="global-header" style="background-image: url('https://techne316.github.io/img/henry-co--odUkx8C2gg-unsplash.jpg.webp'),url('https://techne316.github.io/img/henry-co--odUkx8C2gg-unsplash.jpg');">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <div class="block">
                    <h1>Microarray</h1>
                    <p></p>
                </div>
            </div>
        </div>
    </div>
</section>
    
  </head>
  <body>
    <div class="container">
        <div class="row">
          <div class = "col-md-2"></div>
          <div class = "col-md-9">
            <div class="block">
            <main>

<h1 id="1-example-data">1. Example data</h1>

<p><br></p>

<h3 id="1-project-information"><strong>1) Project Information</strong></h3>

<table class="table table-bordered" style="width: auto !important; ">
<tbody>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> cilent_ID </td>
   <td style="text-align:left;"> Name1 </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> Company.Institution </td>
   <td style="text-align:left;"> company1 </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> Order_Number </td>
   <td style="text-align:left;"> PJ1507 </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> Order_type </td>
   <td style="text-align:left;"> Microarray </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> Species </td>
   <td style="text-align:left;"> Homo sapiens </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> Reference </td>
   <td style="text-align:left;"> UCSC hg19 </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> Array.type </td>
   <td style="text-align:left;"> HG-U133_Plus_2 (Affymetrix Human Genome U133 Plus 2.0 Array) </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;background-color: #E6E6E6 !important;"> Comment </td>
   <td style="text-align:left;"> Basic + DEG </td>
  </tr>
</tbody>
</table>

<p><br><br></p>

<h3 id="2-sample-information"><strong>2) Sample Information</strong></h3>

<p><br></p>

<table class="table table-striped table-condensed" style="width: auto !important; ">
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:left;">   </th>
   <th style="text-align:left;"> sampleID </th>
   <th style="text-align:left;"> status </th>
   <th style="text-align:left;"> filepath </th>
   <th style="text-align:left;"> DEG_group </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> A1 </td>
   <td style="text-align:left;width: 2cm; "> 1 </td>
   <td style="text-align:left;width: 3cm; "> A1 </td>
   <td style="text-align:left;width: 8cm; "> ctrl </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_A.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #E67E22 !important;">ctrl</span> </td>
  </tr>
  <tr>
   <td style="text-align:left;"> A2 </td>
   <td style="text-align:left;width: 2cm; "> 2 </td>
   <td style="text-align:left;width: 3cm; "> A2 </td>
   <td style="text-align:left;width: 8cm; "> ctrl </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_B.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #E67E22 !important;">ctrl</span> </td>
  </tr>
  <tr>
   <td style="text-align:left;"> A3 </td>
   <td style="text-align:left;width: 2cm; "> 3 </td>
   <td style="text-align:left;width: 3cm; "> A3 </td>
   <td style="text-align:left;width: 8cm; "> ctrl </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_C.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #E67E22 !important;">ctrl</span> </td>
  </tr>
  <tr>
   <td style="text-align:left;"> A4 </td>
   <td style="text-align:left;width: 2cm; "> 4 </td>
   <td style="text-align:left;width: 3cm; "> A4 </td>
   <td style="text-align:left;width: 8cm; "> ctrl </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_D.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #E67E22 !important;">ctrl</span> </td>
  </tr>
  <tr>
   <td style="text-align:left;"> B1 </td>
   <td style="text-align:left;width: 2cm; "> 5 </td>
   <td style="text-align:left;width: 3cm; "> B1 </td>
   <td style="text-align:left;width: 8cm; "> cancer </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_E.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #229954 !important;">cancer</span> </td>
  </tr>
  <tr>
   <td style="text-align:left;"> B2 </td>
   <td style="text-align:left;width: 2cm; "> 6 </td>
   <td style="text-align:left;width: 3cm; "> B2 </td>
   <td style="text-align:left;width: 8cm; "> cancer </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_F.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #229954 !important;">cancer</span> </td>
  </tr>
  <tr>
   <td style="text-align:left;"> B3 </td>
   <td style="text-align:left;width: 2cm; "> 7 </td>
   <td style="text-align:left;width: 3cm; "> B3 </td>
   <td style="text-align:left;width: 8cm; "> cancer </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_G.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #229954 !important;">cancer</span> </td>
  </tr>
  <tr>
   <td style="text-align:left;"> B4 </td>
   <td style="text-align:left;width: 2cm; "> 8 </td>
   <td style="text-align:left;width: 3cm; "> B4 </td>
   <td style="text-align:left;width: 8cm; "> cancer </td>
   <td style="text-align:left;width: 3cm; "> ./Rawdata/64969178_H.CEL.gz </td>
   <td style="text-align:left;"> <span style=" font-weight: bold;    color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #229954 !important;">cancer</span> </td>
  </tr>
</tbody>
</table>

<p><br>
<br><br></p>

<h1 id="2-reading-raw-file-and-pre-processing">2. Reading raw file and pre-processing</h1>

<p>Microarray 분석에 앞서, 기본적인 pre-processing을 수행합니다. Affymetrix array chip을 사용하여 분석한 array의 경우 Pre-processing 과정으로는 Background correction, Normalization, Probe specific background correction(PM-correctionrus), Summarization 과정을 통해 진행합니다.</p>

<p>본 분석에서는 아래와 같은 방법으로 normalization을 진행하였습니다.</p>

<ul>
<li>Background correction : &ldquo;rma&rdquo; (Robust Multi-array Average)</li>
<li>Normalization : &ldquo;quantile&rdquo;</li>
</ul>

<p><br></p>

<p><img src="/img/Portfolio_array/boxplot_rawdata-1.png" style="display: block; margin: auto;" /></p>

<p><br></p>

<p><img src="/img/Portfolio_array/boxplot_normalized-1.png" style="display: block; margin: auto;" /></p>

<p><br><br><br>
<br></p>

<h1 id="3-clustering">3. Clustering</h1>

<p>샘플간의 유사성을 확인해보기 위해서 군집 분석을 진행해보도록 하겠습니다. 군집분석에는 여러가지가 있지만, 그 중에서 MDS(Multi-Dimensional Scaling) plot, K-mean clustering을 이용하여 군집이 어떻게 묶이는 지를 확인하고,  Hierarchical clustering를 사용하여 계층간의 유사도도 확인해보도록 하겠습니다.
<br><br></p>

<h3 id="3-1-mds-plot">3.1 MDS plot</h3>

<p>MDS plot (Multi-Dimensional Scaling plot)은 샘플간의 발현량의 차이를 <code>Euclidean distance</code>를 통해 계산해 내고 이를 2차원의 거리로 나타내어 plot한 것입니다. limma package의 <code>plotMDS()</code>를 사용하여 간편하게 나타낼 수 있으며, 특징적으로 가장 차이가 많이 나는 top probe 500개를 선택하여 이에 대한 거리를 계산하였습니다.</p>

<p><img src="/img/Portfolio_array/plotMDS-1.png" width="80%" style="display: block; margin: auto;" /></p>

<p><br><br><br></p>

<h3 id="3-2-k-mean-clustering">3.2 K-mean clustering</h3>

<p>K-mean clustering 방법은 Unsupervised clustering방법 중 하나로, 군집의 갯수($k$)를 지정하고 군집의 갯수에 맞게 비슷한 특성을 가진 데이터를 묶어줍니다. 본 분석에는 K-mean clustering으로 군집은 2개로 지정을 해주었고, random set에서 1개를 고르고 (nstart = 1, default) 최대 iterations 값은 10 (iter.max = 10, default)로 설정해주었습니다.해당하는 군집을 plot으로 표현해주기 위해서 PCA(principal components analysis) plot을 그렸습니다.</p>

<p>PCA plot은 가장 분산이 큰 순서 순으로 PC(principle component)를 정하며 PC1, PC2를 축으로 하여 scatter plot을 그린 것입니다. 위의 <code>plotMDS()</code>와는 다르게 모든 probe를 고려하여 계산하였고, K-mean clustering을 통해서 나온 결과를 검은 색 원형으로 나타내었습니다.</p>

<p>K-mean clustering 결과는 다음과 같습니다.</p>

<ul>
<li>cluster1 : A2, B3</li>
<li>cluster2 : A1, A3, A4, B1, B2, B4</li>
</ul>

<p><img src="/img/Portfolio_array/pca_plot-1.png" width="80%" style="display: block; margin: auto;" />
<br><br>
<br></p>

<h3 id="3-3-hierarchical-clustering">3.3 Hierarchical clustering</h3>

<p>계층적 군집화(Hierarchical clustering)방법은 각 샘플을 순차적으로, 계층적으로 유사한 그룹으로 군집화를 하는 방법입니다.</p>

<ul>
<li>distance method = euclidean <br /></li>
<li>hcluster method = average <br />
<br>
<br>
<img src="/img/Portfolio_array/hclust-1.png" width="80%" style="display: block; margin: auto;" /></li>
</ul>

<p><br><br><br></p>

<h1 id="4-differential-expression-gene-analysis">4. Differential Expression Gene analysis</h1>

<h2 id="4-1-deg-analysis-using-limma">4.1 DEG analysis using limma</h2>

<p>limma는 gene expression microarray data의 분석을 위한 R package로서, linear model를 통한 DEG(differential expression gene analysis) 분석에 특화되어 있습니다.</p>

<p>DEG 분석에 앞서서 PJ1507에 발현랑에 대한 데이터를 살펴보도록 하겠습니다.
먼저, 원본의 데이터는 총 개의 probe set를 모두 이용하여 그룹별 차이에 대한 data table을 그려보도록 하겠습니다.</p>

<p>limma를 이용한 분석 결과에는 여러가지 통계적 지표가 결과값으로 함께 도출됩니다.</p>

<p>&#8226; <strong>logFC</strong> : log2 fold changes (이미 log값이 취해진 경우,  간단하게 그룹의 평균의 차 mean(GrpB) - mean(GrpA))<br />
&#8226; <strong>AveExpr</strong> : average log2-expression level for all samples (모든샘플의 평균 발현량)<br />
&#8226; <strong>t</strong> : moderated t-statistic <br />
&#8226; <strong>P.value</strong> : p-value (t-test의 검정량, 0.05 미만이면 유의미함)<br />
&#8226; <strong>adj.P.Val</strong> :  p-value adjusted for multiple testing (FDR, 0.1 or 0.05 미만이면 유의미함)<br />
&#8226; <strong>B</strong> : B-statistic, log-odds that the gene is differentially expressed<br />
<br></p>

<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:7in; overflow-x: scroll; width:9in; "><table class="table table-striped" style="width: auto !important; ">
<caption>DEG results using limma</caption>
 <thead>
  <tr>
   <th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;"> PROBEID </th>
   <th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;"> SYMBOL </th>
   <th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;"> GENENAME </th>
   <th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;"> logFC </th>
   <th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;"> AveExpr </th>
   <th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;"> t </th>
   <th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;"> P.Value </th>
   <th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;"> adj.P.Val </th>
   <th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;"> B </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> 216919_at </td>
   <td style="text-align:left;"> TP53I11 </td>
   <td style="text-align:left;"> tumor protein p53 inducible protein 11 </td>
   <td style="text-align:right;"> -3.306536 </td>
   <td style="text-align:right;"> 10.69056 </td>
   <td style="text-align:right;"> -5.337927 </td>
   <td style="text-align:right;"> 0.0000381 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.488091 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 1563596_at </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:right;"> 3.075037 </td>
   <td style="text-align:right;"> 11.08042 </td>
   <td style="text-align:right;"> 4.775424 </td>
   <td style="text-align:right;"> 0.0001329 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.498725 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 213899_at </td>
   <td style="text-align:left;"> METAP2 </td>
   <td style="text-align:left;"> methionyl aminopeptidase 2 </td>
   <td style="text-align:right;"> -3.170673 </td>
   <td style="text-align:right;"> 10.96791 </td>
   <td style="text-align:right;"> -4.771291 </td>
   <td style="text-align:right;"> 0.0001342 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.498809 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 238581_at </td>
   <td style="text-align:left;"> GBP5 </td>
   <td style="text-align:left;"> guanylate binding protein 5 </td>
   <td style="text-align:right;"> -2.869328 </td>
   <td style="text-align:right;"> 11.12859 </td>
   <td style="text-align:right;"> -4.581689 </td>
   <td style="text-align:right;"> 0.0002057 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.502748 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 1556536_at </td>
   <td style="text-align:left;"> KIAA2012-AS1 </td>
   <td style="text-align:left;"> KIAA2012 antisense RNA 1 </td>
   <td style="text-align:right;"> 2.907880 </td>
   <td style="text-align:right;"> 10.87613 </td>
   <td style="text-align:right;"> 4.553123 </td>
   <td style="text-align:right;"> 0.0002194 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.503357 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 237154_at </td>
   <td style="text-align:left;"> HSD11B1-AS1 </td>
   <td style="text-align:left;"> HSD11B1 antisense RNA 1 </td>
   <td style="text-align:right;"> 2.761744 </td>
   <td style="text-align:right;"> 11.06853 </td>
   <td style="text-align:right;"> 4.390801 </td>
   <td style="text-align:right;"> 0.0003169 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.506897 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 201501_s_at </td>
   <td style="text-align:left;"> GRSF1 </td>
   <td style="text-align:left;"> G-rich RNA sequence binding factor 1 </td>
   <td style="text-align:right;"> 2.789668 </td>
   <td style="text-align:right;"> 11.13008 </td>
   <td style="text-align:right;"> 4.388571 </td>
   <td style="text-align:right;"> 0.0003185 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.506947 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 222219_s_at </td>
   <td style="text-align:left;"> TLE6 </td>
   <td style="text-align:left;"> TLE family member 6, subcortical maternal complex member </td>
   <td style="text-align:right;"> -2.983176 </td>
   <td style="text-align:right;"> 10.86861 </td>
   <td style="text-align:right;"> -4.369867 </td>
   <td style="text-align:right;"> 0.0003324 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.507364 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 240760_at </td>
   <td style="text-align:left;"> CDRT15 </td>
   <td style="text-align:left;"> CMT1A duplicated region transcript 15 </td>
   <td style="text-align:right;"> 2.845493 </td>
   <td style="text-align:right;"> 11.08404 </td>
   <td style="text-align:right;"> 4.284317 </td>
   <td style="text-align:right;"> 0.0004037 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.509293 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 232905_at </td>
   <td style="text-align:left;"> GTF2H5 </td>
   <td style="text-align:left;"> general transcription factor IIH subunit 5 </td>
   <td style="text-align:right;"> 2.703105 </td>
   <td style="text-align:right;"> 10.81765 </td>
   <td style="text-align:right;"> 4.277430 </td>
   <td style="text-align:right;"> 0.0004101 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.509450 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 217302_at </td>
   <td style="text-align:left;"> OR2F2 </td>
   <td style="text-align:left;"> olfactory receptor family 2 subfamily F member 2 </td>
   <td style="text-align:right;"> -2.859838 </td>
   <td style="text-align:right;"> 11.17119 </td>
   <td style="text-align:right;"> -4.218966 </td>
   <td style="text-align:right;"> 0.0004684 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.510792 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 204357_s_at </td>
   <td style="text-align:left;"> LIMK1 </td>
   <td style="text-align:left;"> LIM domain kinase 1 </td>
   <td style="text-align:right;"> 2.756104 </td>
   <td style="text-align:right;"> 11.27315 </td>
   <td style="text-align:right;"> 4.174953 </td>
   <td style="text-align:right;"> 0.0005178 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.511813 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 228812_at </td>
   <td style="text-align:left;"> REL </td>
   <td style="text-align:left;"> REL proto-oncogene, NF-kB subunit </td>
   <td style="text-align:right;"> 2.688652 </td>
   <td style="text-align:right;"> 11.05191 </td>
   <td style="text-align:right;"> 4.153735 </td>
   <td style="text-align:right;"> 0.0005434 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.512309 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 214826_at </td>
   <td style="text-align:left;"> PDE12 </td>
   <td style="text-align:left;"> phosphodiesterase 12 </td>
   <td style="text-align:right;"> 2.860442 </td>
   <td style="text-align:right;"> 11.06622 </td>
   <td style="text-align:right;"> 4.146743 </td>
   <td style="text-align:right;"> 0.0005521 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.512473 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 230499_at </td>
   <td style="text-align:left;"> BIRC3 </td>
   <td style="text-align:left;"> baculoviral IAP repeat containing 3 </td>
   <td style="text-align:right;"> -2.658312 </td>
   <td style="text-align:right;"> 11.01319 </td>
   <td style="text-align:right;"> -4.146421 </td>
   <td style="text-align:right;"> 0.0005525 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.512481 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 227034_at </td>
   <td style="text-align:left;"> SOWAHC </td>
   <td style="text-align:left;"> sosondowah ankyrin repeat domain family member C </td>
   <td style="text-align:right;"> 2.637866 </td>
   <td style="text-align:right;"> 10.86600 </td>
   <td style="text-align:right;"> 4.144102 </td>
   <td style="text-align:right;"> 0.0005555 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.512535 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 202393_s_at </td>
   <td style="text-align:left;"> KLF10 </td>
   <td style="text-align:left;"> Kruppel like factor 10 </td>
   <td style="text-align:right;"> -2.725271 </td>
   <td style="text-align:right;"> 10.95375 </td>
   <td style="text-align:right;"> -4.133962 </td>
   <td style="text-align:right;"> 0.0005684 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.512774 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 1561923_a_at </td>
   <td style="text-align:left;"> SF3B6 </td>
   <td style="text-align:left;"> splicing factor 3b subunit 6 </td>
   <td style="text-align:right;"> -2.526550 </td>
   <td style="text-align:right;"> 11.18205 </td>
   <td style="text-align:right;"> -4.099594 </td>
   <td style="text-align:right;"> 0.0006147 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.513586 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 202297_s_at </td>
   <td style="text-align:left;"> RER1 </td>
   <td style="text-align:left;"> retention in endoplasmic reticulum sorting receptor 1 </td>
   <td style="text-align:right;"> 2.809862 </td>
   <td style="text-align:right;"> 11.12259 </td>
   <td style="text-align:right;"> 4.081202 </td>
   <td style="text-align:right;"> 0.0006411 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.514022 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 206235_at </td>
   <td style="text-align:left;"> LIG4 </td>
   <td style="text-align:left;"> DNA ligase 4 </td>
   <td style="text-align:right;"> 2.815610 </td>
   <td style="text-align:right;"> 10.89220 </td>
   <td style="text-align:right;"> 4.067505 </td>
   <td style="text-align:right;"> 0.0006614 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.514349 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 241877_at </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:right;"> 2.630619 </td>
   <td style="text-align:right;"> 11.24192 </td>
   <td style="text-align:right;"> 4.036526 </td>
   <td style="text-align:right;"> 0.0007098 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.515091 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 216649_at </td>
   <td style="text-align:left;"> RREB1 </td>
   <td style="text-align:left;"> ras responsive element binding protein 1 </td>
   <td style="text-align:right;"> 2.551856 </td>
   <td style="text-align:right;"> 10.87940 </td>
   <td style="text-align:right;"> 3.976183 </td>
   <td style="text-align:right;"> 0.0008145 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.516549 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 231667_at </td>
   <td style="text-align:left;"> SLC39A5 </td>
   <td style="text-align:left;"> solute carrier family 39 member 5 </td>
   <td style="text-align:right;"> -2.681910 </td>
   <td style="text-align:right;"> 11.01663 </td>
   <td style="text-align:right;"> -3.974787 </td>
   <td style="text-align:right;"> 0.0008171 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.516583 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 219164_s_at </td>
   <td style="text-align:left;"> ATG2B </td>
   <td style="text-align:left;"> autophagy related 2B </td>
   <td style="text-align:right;"> -2.526807 </td>
   <td style="text-align:right;"> 10.55692 </td>
   <td style="text-align:right;"> -3.962426 </td>
   <td style="text-align:right;"> 0.0008404 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.516884 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 228564_at </td>
   <td style="text-align:left;"> LINC01116 </td>
   <td style="text-align:left;"> long intergenic non-protein coding RNA 1116 </td>
   <td style="text-align:right;"> 2.653101 </td>
   <td style="text-align:right;"> 10.80074 </td>
   <td style="text-align:right;"> 3.955181 </td>
   <td style="text-align:right;"> 0.0008544 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.517061 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 222568_at </td>
   <td style="text-align:left;"> UGGT1 </td>
   <td style="text-align:left;"> UDP-glucose glycoprotein glucosyltransferase 1 </td>
   <td style="text-align:right;"> 2.510771 </td>
   <td style="text-align:right;"> 11.13544 </td>
   <td style="text-align:right;"> 3.930278 </td>
   <td style="text-align:right;"> 0.0009043 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.517671 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 236410_x_at </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:right;"> -2.595133 </td>
   <td style="text-align:right;"> 10.76941 </td>
   <td style="text-align:right;"> -3.926993 </td>
   <td style="text-align:right;"> 0.0009111 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.517752 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 1564787_at </td>
   <td style="text-align:left;"> ELMOD1 </td>
   <td style="text-align:left;"> ELMO domain containing 1 </td>
   <td style="text-align:right;"> 2.744767 </td>
   <td style="text-align:right;"> 10.94447 </td>
   <td style="text-align:right;"> 3.914711 </td>
   <td style="text-align:right;"> 0.0009370 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.518054 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 205534_at </td>
   <td style="text-align:left;"> PCDH7 </td>
   <td style="text-align:left;"> protocadherin 7 </td>
   <td style="text-align:right;"> 2.512807 </td>
   <td style="text-align:right;"> 11.22986 </td>
   <td style="text-align:right;"> 3.900839 </td>
   <td style="text-align:right;"> 0.0009671 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.518396 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> 216506_x_at </td>
   <td style="text-align:left;"> MLLT10 </td>
   <td style="text-align:left;"> MLLT10 histone lysine methyltransferase DOT1L cofactor </td>
   <td style="text-align:right;"> 2.662868 </td>
   <td style="text-align:right;"> 10.64324 </td>
   <td style="text-align:right;"> 3.851943 </td>
   <td style="text-align:right;"> 0.0010812 </td>
   <td style="text-align:right;"> 0.9784561 </td>
   <td style="text-align:right;"> -4.519609 </td>
  </tr>
</tbody>
</table><</div>

<p><br><br></p>

<h2 id="4-2-histogram-of-logfc">4.2 Histogram of logFC</h2>

<p>logFC에 대한 차이를 살펴보고자 합니다.</p>

<p>일반적으로 logFC &gt; 1.5 이면 Up-regulated gene 으로 보고, logFC &lt; -1.5이면 Down-regulated gene으로 보고 있습니다. 하지만, 연구에 따라 해당하는 값이 많이 적어 발현량의 차이를 보기 힘든 경우에는 1을 기준으로 보기도 합니다.</p>

<p><img src="/img/Portfolio_array/DEG_logFC_hist-1.png" width="50%" style="display: block; margin: auto;" /></p>

<p>PJ1507의 그룹별로 계산한 logFC의 분포에 따르면,
<code>|logFC| &gt; 1.0</code> 인 probe의 갯수는 10171이며, <code>|logFC| &gt; 1.5</code> 인 probe 갯수는 2603개 입니다. 따라서, 이후 분석되는 DEG set로는 logFC와 P.Value, adj.P.Value를 기준으로 해서 아래와 같이 두개의 다른 DEG set으로 분석을 진행해 보고자 합니다.</p>

<ul>
<li>DEG set1 : |logFC| &gt; 1.0 이고, P.Value &lt; 0.05</li>
<li>DEG set2 : |logFC| &gt; 1.5 이고, adj.P.Value &lt; 0.1
<br><br><br></li>
</ul>

<h2 id="4-3-robust-deg-set1">4.3 Robust DEG set1</h2>

<p>여기서는 <strong>1) |logFC| &gt; 1.0</strong> 이면서, 2) <strong>P.value &lt; 0.05</strong> 인 DEG set1을 만들어 보도록 하겠습니다.</p>

<p><img src="/img/Portfolio_array/DEGset1_volcanoplot-1.png" width="50%" style="display: block; margin: auto;" /></p>

<p>총 2461개의 유의적인 probe set가 나왔으며, 이에 대한 heatmap은 다음과 같습니다.</p>

<ul>
<li>Up-regulated gene : 1254</li>
<li>Down-regulated gene : 1207
<br><br><br>
<img src="/img/Portfolio_array/heatmap_DEG1-1.png" width="120%" />
<br><br><br></li>
</ul>

<h2 id="4-4-strict-deg-set2">4.4 Strict DEG set2</h2>

<p>여기서는 <strong>1) |logFC| &gt; 1.5</strong> 이면서, <strong>2) adj.P.Val &lt; 0.1</strong> 인 DEG set2을 만들어 보도록 하겠습니다.</p>

<p><img src="/img/Portfolio_array/DEGset2_volcanoplot-1.png" width="50%" style="display: block; margin: auto;" /></p>

<p>총 2128개의 유의적인 probe set가 나왔으며, 이에 대한 heatmap은 다음과 같습니다.</p>

<ul>
<li>Up-regulated gene : 1079</li>
<li>Down-regulated gene : 1049
<br><br><br>
<img src="/img/Portfolio_array/heatmap_DEG2-1.png" width="120%" />
<br><br><br></li>
</ul>

<h1 id="5-functional-analysis">5. Functional Analysis</h1>

<h2 id="5-1-gsea-analysis-report">5.1 GSEA Analysis Report</h2>

<p>GSEA (Gene Set Enrichment Analysis)는 두 개의 phenotype을 비교하여 통계적으로 유의성을 가지는 gene sets을 분석해주는 방법입니다. GSEA는 permutation stat을 기본으로 하고 있으며, 각 phenotype에 해당하는 샘플의 갯수가 7개 미만의 경우에는 permutation type을 gene_set으로 하는 것을 추천하고 있습니다.</p>

<p>GSEA에서 분석할 수 있는 Molecular Signatures Database로는 아래의 8항목이 있습니다.</p>

<p>&#8226; H: hallmark gene sets<br />
&#8226; C1: positional gene sets<br />
&#8226; C2: curated gene sets<br />
&#8226; C3: motif gene sets<br />
&#8226; C4: computational gene sets<br />
&#8226; C5: GO gene sets<br />
&#8226; C6: oncogenic signatures<br />
&#8226; C7: immunologic signatures</p>

<p><br>
따라서, GSEA 분석에는 아래와 같은 parameter로서 분석을 진행하였습니다.</p>

<p>&#9658; <strong>Permutation type</strong> : gene_set<br />
&#9658; <strong>Number of permutations</strong> : 1000<br />
&#9658; <strong>Gene sets database</strong> : Hallmark gene set 7.0</p>

<p><br><br></p>

<h3 id="gsea-analysis-results-in-grpb-cancer">GSEA analysis results in GrpB (&ldquo;cancer&rdquo;)</h3>

<p>&#8226; Total 4 samples <br />
&#8226; 27 gene sets are upregulated in phenotype cancer <br />
&#8226; 0 gene sets are significant at FDR &lt; 25% <br />
&#8226; 1 gene sets are significantly enriched at nominal pvalue &lt; 1% <br />
&#8226; 1 gene sets are significantly enriched at nominal pvalue &lt; 5% <br />
<br><br>
<img src="/img/Portfolio_array/GSEAplot_grpB-1.png" alt="" /><!-- -->
<br><br></p>

<p>Among the several gene sets,  HALLMARK_UV_RESPONSE_UP is a top up-regulated gene sets in cancer
<br><br>
<div class="figure" style="text-align: center">
<iframe src="/img/Portfolio_array/PJ1507.HALLMARK_UV_RESPONSE_UP.plot.cancer.1.pdf" alt="GrpB top NES gene set" scrolling="no" style="width: 100%;height: 100% ;border: none;" onload="this.style.height=(this.contentDocument.body.scrollHeight+500) +'px';"></iframe></p>

<p><p class="caption">GrpB top NES gene set</p>
</div>
<br><br><br></p>

<h3 id="gsea-analysis-results-in-grpa-ctrl">GSEA analysis results in GrpA (&ldquo;ctrl&rdquo;)</h3>

<p>&#8226; Total 4 samples <br />
&#8226; 23 gene sets are upregulated in phenotype ctrl <br />
&#8226; 0 gene sets are significant at FDR &lt; 25% <br />
&#8226; 0 gene sets are significantly enriched at nominal pvalue &lt; 1% <br />
&#8226; 0 gene sets are significantly enriched at nominal pvalue &lt; 5% <br />
<br><br>
<img src="/img/Portfolio_array/GSEAplot_grpA-1.png" alt="" /><!-- -->
<br />
Among the several gene sets,  HALLMARK_INTERFERON_ALPHA_RESPONSE is a top up-regulated gene sets in ctrl
<br><br>
<div class="figure" style="text-align: center">
<embed src="/img/Portfolio_array/PJ1507.HALLMARK_INTERFERON_ALPHA_RESPONSE.plot.ctrl.1.pdf" alt="GrpA top NES gene set"  />
<p class="caption">GrpA top NES gene set</p>
</div>
<br><br><br></p>

<h3 id="gsea-leading-edge-analysis">GSEA leading edge analysis</h3>

<div class="figure">
<embed src="/img/Portfolio_array/PJ1507.leading.overlap.cancer.pdf" alt="GSEA_leading edge analysis" width="50%" /><img src="/img/Portfolio_array/PJ1507.leading.overlap.ctrl.pdf" alt="GSEA_leading edge analysis" width="50%" />
<p class="caption">GSEA_leading edge analysis</p>
</div>
</main>
            </div>
          </div>
        </div>    
    </div>  
  </body>
</html>

	

	
		
<section id="call-to-action" style="background-image: url('https://techne316.github.io/img/call-to-action-bg.jpg');">
  
  <div class="container">
    <div class="row">
      <div class ="col-md-3"></div>
      <div class="col-md-12">
        <div class="block">
          <h2>We design delightful digital experiences.</h2>
          <p>Read more about what we do and our philosophy of design. Judge for yourself The work and results we’ve achieved for other clients, and meet our highly experienced Team who just love to design.</p>
          
          <a class="btn btn-main btn-solid-border" href="#" >tell us your story</a>
          
        </div>
      </div>
      <div class="col-md-3"></div>
    </div>
  </div>
</section>

	

	
<footer>
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <div class="footer-manu">
          <ul>
            
            <li><a href="#">About us</a></li>
            
            <li><a href="#">Contact us</a></li>
            
            <li><a href="#">How it works</a></li>
            
            <li><a href="#">Blog</a></li>
            
            <li><a href="#">Terms</a></li>
            
          </ul>
        </div>
        <p>Copyright &copy; Design &amp; Developed by Techne. All rights reserved.</p>
      </div>
    </div>
  </div>
</footer>


	
</body>
</html>